packages/document/docs/zh/plugin/official-plugins/shiki.mdx

```diff
@@ -64,81 +66,34 @@ Transformer 是本插件中的一个概念，它的作用是对代码块的特
 
 本插件中内置了一些 Transformer，包括：
 
-- `createTransformerDiff`：实现代码块的 diff 高亮效果。
 - `createTransformerLineNumber`：实现代码块的行号显示。
-- `createTransformerErrorLevel`：实现代码块对应行的错误等级显示，包括 `error` 和 `warning`。
-- `createTransformerHighlight`：实现代码块的行高亮显示。
-- `createTransformerFocus`： 实现代码块的行聚焦显示。
 
 你可以通过配置 `transformers` 属性来启用这些 Transformer，比如：
 
 ```ts title="rspress.config.ts"
 import { defineConfig } from 'rspress/config';
 import { pluginShiki, createTransformerDiff } from '@rspress/plugin-shiki';
+import {
+  transformerNotationDiff,
+  transformerNotationErrorLevel,
+  transformerNotationFocus,
+  transformerNotationHighlight,
+} from '@shikijs/transformers';
 
 export default defineConfig({
   plugins: [
     pluginShiki({
       transformers: [
         // 按需加入即可
-        createTransformerDiff(),
-        // createTransformerLineNumber(),
-        // createTransformerErrorLevel(),
-        // createTransformerHighlight(),
-        // createTransformerFocus(),
+        createTransformerLineNumber(),
+        // transformerNotationDiff(),
+        // transformerNotationErrorLevel(),
+        // transformerNotationHighlight(),
+        // transformerNotationFocus(),
       ],
     }),
   ],
 });
 ```
 
-接着我们来介绍一下如何使用这些 Transformer 对应的语法。
```

Comment:
We can not remove exports in this plugin, you should reexport in this plugin and keep all docs here to avoid breaking change.

Furthermore, we can give an example in docs to introduce how to import other transformers in this plugin.

Reply:
The transformers signatures are changed, so I believe we should make a breaking change for this plugin.

> Furthermore, we can give an example in docs to introduce how to import other transformers in this plugin.

https://github.com/JounQin/rspress/blob/e11037c06d110575bda056f9a8200049d7d303b2/packages/document/docs/zh/plugin/official-plugins/shiki.mdx?plain=1#L71-L97

Reply:
breaking change for now is not acceptable, and we can make this breaking change in Rspress 2.0 since all core and package are using same version for now.

Reply:
If you want a canary version now, I can release for you for this plugin.

Reply:
I don't quite understand, all official plugin major versions are bound with rspress core? That would make plugin evolution too slow maybe.

Reply:
Oh it seems the release workflow can not run in a forked repo, I think you can just patch this plugin using `pnpm patch` or other tools in package manager level.

Reply:
Hah, I mean to copy the codes into our own repo as a thirty-party plugin, I'm building a company level document tool, so local patch won't work.

Reply:
Got it, this package is MIT license, just do it.